app/helpers/SessionAnswersHelper.scala

```diff
@@ -209,4 +211,39 @@ object SessionAnswersHelper extends ImplicitDateFormatter {
       }
     }
   }
+
+  def getContentForAgentsCheckYourAnswersPage()(implicit request: Request[_], messages: Messages): Seq[(String, String, String)] = {
+
+    val seqWhoPlannedToSubmitVATReturn = Seq(
+      (messages("checkYourAnswers.agents.whoPlannedToSubmitVATReturn"),
+        messages(s"checkYourAnswers.agents.whoPlannedToSubmitVATReturn.${request.session.get(SessionKeys.whoPlannedToSubmitVATReturn).get}"),
+        controllers.routes.AgentsController.onPageLoadForWhoPlannedToSubmitVATReturn(CheckMode).url))
+
+    val seqWhyWasTheReturnSubmittedLate = if(request.session.get(SessionKeys.whoPlannedToSubmitVATReturn).get.equals("agent")) {
+      Seq((messages("checkYourAnswers.agents.whyWasTheReturnSubmittedLate"),
+        messages(s"checkYourAnswers.agents.whyWasTheReturnSubmittedLate.${request.session.get(SessionKeys.causeOfLateSubmissionAgent).get}"),
+        controllers.routes.AgentsController.onPageLoadForWhyReturnSubmittedLate(CheckMode).url))
+    }
+    else Seq.empty
+
+    seqWhoPlannedToSubmitVATReturn ++ seqWhyWasTheReturnSubmittedLate
+  }
+
+  def getAllTheContentForCheckYourAnswersPage()(implicit request: Request[_], messages: Messages): Seq[(String, String, String)]={
+
+    val reasonableExcuse = request.session.get(SessionKeys.reasonableExcuse).get
+    val agentSession =request.session.get(SessionKeys.agentSessionVrn).isDefined
+
+    if(isAllAnswerPresentForReasonableExcuse(reasonableExcuse) && !agentSession) {
+      getContentForReasonableExcuseCheckYourAnswersPage(reasonableExcuse)
+    }
+      else if(agentSession){
+      getContentForAgentsCheckYourAnswersPage() ++ getContentForReasonableExcuseCheckYourAnswersPage(reasonableExcuse)
+    }
+    else {
+      logger.error(s"[SessionAnswersHelper] User hasn't got all keys in session for reasonable excuses :$reasonableExcuse")
```

Comment:
I don't think you need this extra logging as (in the controller) it is already logged. So I think you can just return the empty Seq.

Reply:
Done